charts/ocis/values.yaml

```diff
@@ -2295,209 +5072,504 @@ services:
         # -- URL to load themes from. Will be prepended to the theme path. Defaults to the value of "externalDomain".
-        server: ""
+        server:
+        # @schema
+        # type: string
+        # required: true
+        # @schema
         # -- URL path to load themes from. The theme server will be prepended. Defaults to the ownCloud Web default theme.
-        path: "/themes/owncloud/theme.json"
+        path: /themes/owncloud/theme.json
+        # @schema
+        # type: string
+        # required: true
+        # @schema
         # -- Name of the theme you provide via `configRefs.webThemeConfigRef` and `configRefs.webThemeAssetsConfigRef`.
         # If you change this when providing a custom theme, you must also change `theme.path`.
-        themeNameConfigRefs: "owncloud"
+        themeNameConfigRefs: owncloud
       oidc:
+        # @schema
+        # type: string
+        # required: true
+        # @schema
         # -- OIDC scopes to request during authentication to authorize access to user details. Defaults to ‘openid profile email’. Values are separated by blank. More example values but not limited to are ‘address’ or ‘phone’ etc.
-        webClientScope: "openid profile email"
+        webClientScope: openid profile email
+        # @schema
+        # type: string
+        # required: true
+        # @schema
         # -- Specify the client ID which the web frontend will use
         webClientID: web
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
         # -- URI where to redirect the user after a logout was performed. Defaults to the URI of the login page.
-        postLogoutRedirectURI: ""
+        postLogoutRedirectURI:
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
         # -- Specifies the target url valid for the logged out / access denied page.
         loginURL:
+      # @schema
+      # type: boolean
+      # required: true
+      # @schema
       # Defines whether one ore more filters must be set in order to list users in the Web admin settings.
       userListRequiresFilter: false
       # Embed mode settings
       embed:
-        # Specifies if web "embed"-mode is enabled. Defaults to not being set (= disabled).
-        enabled: ""
-        # Specifies how web is being integrated when running in "embed"-mode. Currently supported values are "location" and "" (default).
-        target: ""
+        # @schema
+        # type: boolean
+        # required: true
+        # @schema
+        # Enables the ownCloud Web "embed"-mode.
+        enabled: false
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
+        # Specifies how web is being integrated when running in "embed"-mode. Currently supported values are "location" and  (default).
+        target:
+        # @schema
+        # type: [string, null]
+        # required: false
+        # @schema
```

Comment:
We could add `format: "uri"` here.

Reply:
This could also be eg. `*`, which would not be a valid uri